### ISSUE TITLE
16/10/23_NY_IPA_RideSeach_Image is missing from update profile for accessibility_parent-id_1980

### ISSUE THREAD

**Issue by rohanrp179** [nammayatri/nammayatri#3679]
#1980 

Image is missing from update profile for accessibility

![IMG_0151](https://github.com/nammayatri/nammayatri/assets/138564517/f0265546-fc2b-4b73-a1da-5aa8e7dbc795)

**Comment by aryan-13** [comment]
Fixed

**Comment by Rajaprabhu-Natrajan** [comment]
it's working fine
